README.md

```diff
@@ -163,7 +170,11 @@ kubectl -n stackrox exec -i deploy/central -- bash -c 'ROX_ADMIN_PASSWORD=$(cat)
 
 Then install stackrox-secured-cluster-services (with the init bundle you just generated).
 
-If you're installing on a reasonably sized cluster, use the default installation command:
+To perform the installation, choose one of the following commands depending on your cluster size.
+
+#### Default Secured Cluster Services Installation
+
+If you're installing in a reasonably sized cluster, use the default installation command:
```

Comment:
The change here is the 4th word: "on" vs "in".  I realized the headers I had removed used "installation _in_", so I changed this new text to match that.